### PR TITLE
fixjson: init at 1.1.1

### PR DESCRIPTION
**Pull request by cpcloud**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/node-packages/composition.nix [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
+    inherit (pkgs) stdenv python2 utillinux runCommand writeTextFile;
     inherit nodejs;
     libtool = if pkgs.stdenv.isDarwin then pkgs.darwin.cctools else null;
   };
 in
 import ./node-packages.nix {
   inherit (pkgs) fetchurl fetchgit;
   inherit nodeEnv;
-}
+}
```
Comment (SuperSandro2000) [inline pkgs/development/node-packages/composition.nix:17]: Please revert the changes in this file.

File: pkgs/development/node-packages/node-env.nix [hunk 2]
```diff
@@ -516,7 +516,7 @@ let
     stdenv.mkDerivation {
       name = "node-shell-${name}-${version}";
 
-      buildInputs = [ python nodejs ] ++ stdenv.lib.optional (stdenv.isLinux) util-linux ++ buildInputs;
+      buildInputs = [ python nodejs ] ++ stdenv.lib.optional (stdenv.isLinux) utillinux ++ buildInputs;
```
Comment (SuperSandro2000) [inline pkgs/development/node-packages/node-env.nix:519]: Please revert the changes in this file.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/node-packages/composition.nix (hunk 1 above)
Comment (cpcloud) [reply pkgs/development/node-packages/composition.nix]: Done.

File: pkgs/development/node-packages/node-env.nix (hunk 2 above)
Comment (cpcloud) [reply pkgs/development/node-packages/node-env.nix]: Done.

**Review by midchildan — APPROVED** [review]

**Comment by cpcloud** [comment]
@SuperSandro2000 Can this be merged?

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 106004` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>lumo</li>
    <li>tests.writers</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>iosevka</li>
    <li>vscode-extensions.matklad.rust-analyzer</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 106004` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>iosevka</li>
    <li>lumo</li>
  </ul>
</details>
<details>
  <summary>23 packages built:</summary>
  <ul>
    <li>bitwarden-cli</li>
    <li>castnow</li>
    <li>emojione</li>
    <li>epgstation</li>
    <li>flood</li>
    <li>gtop</li>
    <li>image_optim</li>
    <li>joplin</li>
    <li>mirakurun</li>
    <li>netlify-cli</li>
    <li>pm2</li>
    <li>ripcord</li>
    <li>tests.writers</li>
    <li>thelounge</li>
    <li>twemoji-color-font</li>
    <li>vimPlugins.coc-metals</li>
    <li>vimPlugins.coc-prettier</li>
    <li>vimPlugins.coc-rust-analyzer</li>
    <li>vimPlugins.coc-stylelint</li>
    <li>vimPlugins.coc-vetur</li>
    <li>vscode-extensions.matklad.rust-analyzer</li>
    <li>vscode-extensions.ms-python.vscode-pylance</li>
    <li>vscode-extensions.vadimcn.vscode-lldb</li>
  </ul>
</details>
